src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/VulnerabilityNotificationMessageBuilder.java

```diff
@@ -151,9 +152,9 @@ public String getNotificationType() {
 
             LinkableItem vulnIdItem = new LinkableItem(MessageBuilderConstants.LABEL_VULNERABILITIES, vulnerabilityId, vulnerabilityUrl);
             // TODO in Black Duck 2019.10.0, the VulnerabilitySourceQualifiedId includes the severity so we don't need to make a request to get it
-            String notificationSeverity = blackDuckResponseCache.getSeverity(vulnerabilityUrl);
+            ProjectVersionVulnerableBomComponentsItemsVulnerabilityWithRemediationSeverityType notificationSeverity = blackDuckResponseCache.getSeverity(vulnerabilityUrl);
             if (vulnerabilityFilters.isEmpty() || vulnerabilityFilters.contains(notificationSeverity)) {
-                LinkableItem severity = new LinkableItem(MessageBuilderConstants.LABEL_VULNERABILITY_SEVERITY, notificationSeverity);
+                LinkableItem severity = new LinkableItem(MessageBuilderConstants.LABEL_VULNERABILITY_SEVERITY, notificationSeverity.name());
```

Comment:
Based on the `TODO`, it looks like the item the loop is on (`VulnerabilitySourceQualifiedId vulnerabilityItem`) should now have the severity. Can we replace the line getting it from the `blackDuckResponseCache`? If so, let's remove the `TODO`.

Reply:
the object is manually maintained in the API, and does not have a severity field.  do we know who made the TODO/how they heard that this change would happen in 2019.10.0?

Reply:
We would need to search the issue tracker on that version. I'll discuss more offline.